### PR TITLE
[#119] - Preloading the template chunk for the certificate file

### PR DESCRIPTION
**Pull request by sumitnitsurat**
- added the preloading of templates before the verification starts.

### CODE REVIEW

**Comment by rjchow** [comment]
@sumitnitsurat please review and write tests when you get back 😜

File: src/sagas/templates.test.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { put, call } from "redux-saga/effects";
+import { get } from "lodash";
+import { certificateData } from "@govtechsg/open-certificate";
+import templates from "../components/CertificateTemplates";
+import json from "../../tests_fixture/NP_Certs_FT_BMS.json";
```
Comment (rjchow) [inline src/sagas/templates.test.js]: i think just use a minimal cert inline instead of using a full cert

**Review by rjchow — CHANGES_REQUESTED** [review]

File: src/sagas/templates.test.js (hunk 1 above)
Comment (sumitnitsurat) [reply src/sagas/templates.test.js]: We have any function which returns the minimal cert or you want me to remove the data from this cert itself ?

Comment (rjchow) [reply src/sagas/templates.test.js]: you can use this: https://github.com/OpenCerts/opencerts-website/blob/master/src/sagas/testutils.js

**Review by rjchow — APPROVED** [review]

**Pull request merged**